pkg/model/components/clusterautoscaler.go

```diff
@@ -83,6 +83,9 @@ func (b *ClusterAutoscalerOptionsBuilder) BuildOptions(o interface{}) error {
 	if cas.NewPodScaleUpDelay == nil {
 		cas.NewPodScaleUpDelay = fi.String("0s")
 	}
+	if cas.Metrics == nil {
```

Comment:
You already default this one to false in the manifest. So you do not need this bit

Reply:
removed it ^^